README.md

```diff
@@ -56,6 +56,19 @@ The official config loader for [Tableau](https://github.com/tableauio/tableau).
 
 - [Protocol Buffers Go Reference](https://protobuf.dev/reference/go/)
 
+## C#
+
+### Requirements
+
+- dotnet-sdk-8.0
```

Comment:
About C# **language version**, **compiler version**, and **runtime version**: 
1. https://docs.unity3d.com/6000.0/Documentation/Manual/csharp-compiler.html
2. https://docs.unity3d.com/6000.0/Documentation/Manual/dotnet-profile-support.html

We should constrain them accurately.